roles/network_plugin/contiv/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+contiv_cert_dir: "/etc/contiv/certs"
+contiv_netmaster_port: 9999
+contiv_cni_version: 1.0.1
+contiv_k8s_api_server: "https://{{ hostvars[groups['kube-master'][0]]['ansible_default_ipv4']['address'] }}:6443"
+
+# Parameters for Contiv api-proxy
+contiv_enable_api_proxy: true
+contiv_api_proxy_port: 10000
+contiv_generate_certificate: true
+
+# Forwarding mode: bridge or routing
+contiv_fwd_mode: bridge
+
+# Fabric mode: aci, aci-opflex or default
+contiv_fabric_mode: default
+
+# Dataplane interface
+contiv_vlan_interface: ""
+
+# Default network configuration
+contiv_default_subnet: "10.200.0.0/24"
```

Comment:
Shouldn't this be set to the kube_pods_subnet?
https://github.com/kubernetes-incubator/kargo/blob/master/inventory/group_vars/k8s-cluster.yml#L80-L83

The default here only lets you have ~254 pods. That won't be useful for any production deployment.